### PR TITLE
Added meeting notes from 2019.05.14

### PR DESCRIPTION
**Pull request by igaray**

### CODE REVIEW

**Comment by igaray** [comment]
@spastorino

File: working-groups/learning/minutes/agenda.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# WG-Learning future meeting agenda
```
Comment (spastorino) [inline working-groups/learning/minutes/agenda.md]: I'd use a different name for this file that explains better what the title says. Nothing comes to my mind to be honest, maybe a native speaker can help here :).

File: working-groups/learning/minutes/2019.05.14-meeting.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Status:
+
+- Would be good to schedule a lecture on each missing topic
+- less work for the experts
+- Can then transcribe into a chapter
+- @spastorino will arrange lectures, nikomatsakis will create zoom
+
+## How do we want to keep track of who is doing what? Does github Projects works nice for this?
+
```
Comment (spastorino) [inline working-groups/learning/minutes/2019.05.14-meeting.md]: Maybe we also want to link the issues that were opened during the meeting?.

File: working-groups/learning/minutes/agenda.md (hunk 1 above)
Comment (igaray) [reply working-groups/learning/minutes/agenda.md]: "Agenda" is "a list of items to be discussed at a formal meeting". Perhaps `future_discussion_topics.md` ?

File: working-groups/learning/minutes/2019.05.14-meeting.md (hunk 2 above)
Comment (igaray) [reply working-groups/learning/minutes/2019.05.14-meeting.md]: good idea

**Comment by spastorino** [comment]
I think this is fine. Gonna leave it open for a bit in case others want to review.

**Pull request merged**